api/config.py

```diff
@@ -77,7 +77,8 @@
     __config['persistent']['db_uri'],
     j=True,
     connectTimeoutMS=__config['persistent']['db_connect_timeout'],
-    serverSelectionTimeoutMS=__config['persistent']['db_server_selection_timeout']
+    serverSelectionTimeoutMS=__config['persistent']['db_server_selection_timeout'],
+    connect=False
```

Comment:
Ending the last line of such lists with a comma (,) allows for later additions to be one-line changes, unlike it was the case here.